lib/portlayer/exec/base.go

```diff
@@ -261,6 +261,17 @@ func (c *containerBase) startGuestProgram(op trace.Operation, name string, args
 
 	_, err = m.StartProgram(op, &auth, &spec)
 
+	if tasks.IsInvalidPowerStateError(err) {
+		return err
+	}
+
+	if tasks.IsInvalidStateError(err) {
+		// we special case this for now as we _presume_ that it's either a conflict
+		// of reconfigure with guest operation, or a conflict with another guest operation.
+		// These cases do not return a TaskInProgress or concurrent modification.
+		return ConcurrentAccessError{errors.New("invalid state from start guest program")}
```

Comment:
Would there be any benefit to logging `err` or including it in the new error being constructed here? (E.g., to allow us to use log data to validate the presumption described in the comment above if something doesn't seem to be working quite as expected.)